config/packages/easy_admin/conference.yaml

```diff
@@ -48,7 +63,22 @@ easy_admin:
                     - { property: 'cfpEndAt', type: 'date', 'type_options': { 'widget': 'single_text' } }
                     - { type: 'group', columns: 4, icon: 'cubes', label: 'Metadata' }
                     - { property: 'articleUrl', type: 'url' }
-        ExcludedConferences:
+            show:
+                fields:
+                    - { property: name, type: text }
```

Comment:
too many duplication here. Let's use YAML merge operator: https://jolicode.com/blog/loperateur-de-merge-en-yaml-le-pire-ennemi-de-la-duplication-de-configuration

And thanks to it the list will be in the same order on all menu. Consistency is **really** important (in the UI, and in the code too: naming, style, etc)